xml/docu_styleguide.language.xml

```diff
@@ -405,6 +405,49 @@
    </para>
    </sect2>
 
+ <sect2 xml:id="sec-quotation">
+  <title>Quotations</title>
+  <para>
+  Use quotations to quote from sources, such as books.
+  In all other cases, do not use quotation marks:
+  </para>
+  <itemizedlist>
+  <listitem>
+  <para>
+  For computer input, computer output, and user
+  interface elements, use different markup.
+  See also <xref linkend="sec-command-line"/> and
+  <xref linkend="sec-ui-item-markup"/>.
+  </para>
+  </listitem>
+  <listitem>
+  <para>
+  Do not use quotation marks to indicate irony.
+  Avoid irony in technical writing.
+  See also <xref linkend="sec-tone"/>.
```

Comment:
The list here is "unparallel": the first item is a "do this" and the second is a "don't do this". That _could_ be confusing. Not sure how to solve that. :thinking: Can both items be written in parallel?

Reply:
Don't know whether it's worth the effort. The audience are technical writers. We should be able to cope with this level of "complexity" ;)

Reply:
There are only two items, one 'do', the other' don't', that's a parallel already :-)

Reply:
I think the "do not" is very important here.